src/testdir/test_quickfix.vim

```diff
@@ -6462,6 +6462,32 @@ func Test_quickfix_buffer_contents()
   call setqflist([], 'f')
 endfunc
 
+func Test_quickfix_update()
+  " Setup: populate a couple buffers
+  call setline(1, range(1,5))
```

Comment:
You should create a new buffer before changing the contents of the buffer.

Reply:
Done!